src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,33 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        ASSESSMENT = "Assessment"
+        PATCH_INSTALLATION = "PatchInstallation"
+        PACKAGE_MANAGER = "package_manager"
+        NUMBER_OF_TRIALS = "number_of_trials"
+        NUMBER_OF_ROUNDS = "number_of_rounds"
+        FAILED = "failed"
+        SUCCEEDED = "succeeded"
+        ERROR_MSG = "error_msg"
+        INSTALLED_UPDATE_COUNT = "installed_update_count"
+        UPDATE_RUN_SUCCESSFUL = "update_run_successful"
+        MAINTENANCE_WINDOW = "maintenance_window"
+        PERC_MAINTENANCE_WINDOW_USED = "perc_maintenance_window_used"
+        MAINTENANCE_WINDOW_EXCEEDED = "maintenance_window_exceeded"
+        START_TIME = "start_time"
+        END_TIME = "end_time"
+        TIME_TAKEN = "time_taken"
+        MACHINE_INFO = "machine_info"
+        MESSAGE = "message"
+
+    # Stopwatch exception strings
+    STARTED_ALREADY = "Stopwatch is already started" 
+    STOPPED_ALREADY = "Stopwatch is already stoppped"
```

Comment:
Since these are specific to Stopwatch and won't be re-used in other places, add these strings within Stopwatch

Reply:
Done.